client/asset/eth/nodeclient.go

```diff
@@ -207,20 +202,32 @@ func (n *nodeClient) balance(ctx context.Context) (*Balance, error) {
 	}
 
 	outgoing := new(big.Int)
+	incoming := new(big.Int)
+	zero := new(big.Int)
 	for _, tx := range pendingTxs {
 		// TODO: Check that the from address matches addr. types.Transaction
 		// does not expose the from address directly.
-		outgoing.Add(outgoing, tx.Value())
+		gas := new(big.Int).SetUint64(tx.Gas())
+		fees := new(big.Int).Mul(gas, tx.GasFeeCap())
```

Comment:
Should we also handle legacy transactions (`GasPrice` is populated and not `GasFeeCap`)? This would only be an issue if someone imports their seed into another wallet.